installer/scripts/omsadmin.sh

```diff
@@ -310,6 +310,28 @@ set_proxy_setting()
     fi
 }
 
+onboard_scom()
+{
+    echo "Onboarding SCOM"
+    create_workspace_directories "scom"
+    touch $CONF_DIR/omsagent.conf
+    update_path $CONF_DIR/omsagent.conf
+    chown_omsagent $CONF_DIR/*
+    make_dir $CONF_DIR/omsagent.d
+    #Always register SCOM as secondary Workspace
+    echo "SCOM Workspace" > $CONF_DIR/.multihoming_marker
+    configure_logrotate
+
+    # The scx certificate and key needs to be owned by omsagent so that it is readable and can be used during SCOM authentication
+    if [ -f /etc/opt/microsoft/scx/ssl/scx.pem ]; then
```

Comment:
I'm not quite sure why you're checking for a certificate in that location (at least directly). Isn't this historical for older agents (when OMI and SCX were shipping together)?

Since OMS always installs a newer version of SCX, then I'd think that SCX couldn't possibly have a key stored in that location anymore. Why bother checking for that here? When would this ever be necessary?